Palace/Utilities/Localization/TransifexManager.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TransifexManager.swift
+//  Palace
+//
+//  Created by Maurice Carrier on 9/30/22.
+//  Copyright © 2022 The Palace Project. All rights reserved.
+//
+
+import Foundation
+
+struct TransifexManager {
+  static let token: [UInt8] = [101, 127, 97, 112, 66, 71, 38, 7, 15, 7, 80, 2, 76, 92, 125, 103, 42, 86,
```

Comment:
Is it ok that we keep token here and not in secrets?

Reply:
Ah, completely missed TPPSecrets, where is the encoding function, I can reuse the salt to encode the token

Reply:
Ah, I recreated the existing obfuscator. Will update